src/lifx.d.ts

```diff
@@ -403,137 +356,15 @@ export const constants: {
   TAGGED_BIT: number;
   ZONE_INDEX_MAXIMUM_VALUE: number;
   ZONE_INDEX_MINIMUM_VALUE: number;
-};
-
-export namespace packet {
-  const typeList: {
-    id: number;
-    name: string;
-  }[];
-
-  /**
-  * Creates a new packet by the given type
-  * Note: This does not validate the given params
-  * @param  {String|Number} type the type of packet to create as number or string
-  * @param  {Object} params further settings to pass
-  * @param  {String} [source] the source of the packet, length 8
-  * @param  {String} [target] the target of the packet, length 12
-  * @return {Object} The prepared packet object including header
-  */
-  function create(type: any, params: any, source: any, target: any): any;
-
-  /**
-  * Creates a lifx packet header from a given object
-  * @param {Object} obj Object containg header configuration for packet
-  * @return {Buffer} packet header buffer
-  */
-  function headerToBuffer(obj: any): any;
-
-  /**
-  * Parses a lifx packet header
-  * @param {Buffer} buf Buffer containg lifx packet including header
-  * @return {Object} parsed packet header
-  */
-  function headerToObject(buf: any): any;
-
-  /**
-  * Creates a packet from a configuration object
-  * @param {Object} obj Object with configuration for packet
-  * @return {Buffer|Boolean} the packet or false in case of error
-  */
-  function toBuffer(obj: any): any;
-
-  /**
-  * Parses a lifx packet
-  * @param {Buffer} buf Buffer with lifx packet
-  * @return {Object} parsed packet
-  */
-  function toObject(buf: any): any;
 }
 
-export namespace utils {
+export interface utils {
```

Comment:
Please revert this to `namespace` and as it where. I don't think it's a good idea to remove types we've previously published. I might have made a mention on private/public APIs but since the functions in `utils` had JSDoc's, they could be used by some.